include/uv.h

```diff
@@ -613,7 +613,13 @@ enum uv_udp_flags {
    * must not be freed by the recv_cb callback.
    */
   UV_UDP_MMSG_CHUNK = 8,
-
+  /*
+   * Indicates if IP_RECVERR/IPV6_RECVERR will be set when binding the handle.
+   * This sets IP_RECVERR for IPv4 and IPV6_RECVERR for IPv6 UDP sockets on
+   * Linux. This stops the Linux kernel from supressing some ICMP error messages
+   * and enables full ICMP error reporting for faster failover.
+   */
+  UV_UDP_RECVERR = 16,
```

Comment:
This flag should also be added to the [documentation](https://github.com/libuv/libuv/blob/v1.x/docs/src/udp.rst)

Reply:
The flag has been added to documentation.